galleries/examples/statistics/histogram_multihist.py

```diff
@@ -27,24 +27,77 @@
 fig, ((ax0, ax1), (ax2, ax3)) = plt.subplots(nrows=2, ncols=2)
 
 colors = ['red', 'tan', 'lime']
-ax0.hist(x, n_bins, density=True, histtype='bar', color=colors, label=colors)
+ax0.hist(x, n_bins, density=True, histtype="bar", color=colors, label=colors)
 ax0.legend(prop={'size': 10})
 ax0.set_title('bars with legend')
 
-ax1.hist(x, n_bins, density=True, histtype='bar', stacked=True)
+ax1.hist(x, n_bins, density=True, histtype="bar", stacked=True)
 ax1.set_title('stacked bar')
 
-ax2.hist(x, n_bins, histtype='step', stacked=True, fill=False)
+ax2.hist(x, n_bins, histtype="step", stacked=True, fill=False)
 ax2.set_title('stack step (unfilled)')
 
 # Make a multiple-histogram of data-sets with different length.
 x_multi = [np.random.randn(n) for n in [10000, 5000, 2000]]
-ax3.hist(x_multi, n_bins, histtype='bar')
+ax3.hist(x_multi, n_bins, histtype="bar")
 ax3.set_title('different sample sizes')
 
 fig.tight_layout()
 plt.show()
 
+# %%
+# Setting properties for each sample set
+# ------------
```

Comment:
It looks like Sphinx is complaining that the underline here should be the same length as the title
https://app.circleci.com/pipelines/github/matplotlib/matplotlib/31277/workflows/23056dd7-aedd-4086-b926-b84af3dbcbc3/jobs/83225?invite=true#step-114-0_109

Reply:
Hmmm, is there a place where these warnings are documented? Or is it hidden somewhere in the CircleCI debug info?

Reply:
yep, you are right